### PR TITLE
feat(db) add tagging support for credentials

### PR DESCRIPTION
**Pull request by hbagdi**

### CODE REVIEW

**Comment by hbagdi** [comment]
These migrations are incorrect. They need to be in the plugin migrations.

File: kong/plugins/acl/daos.lua [hunk 1]
```diff
@@ -12,6 +12,7 @@ return {
       { created_at = typedefs.auto_timestamp_s },
       { consumer = { type = "foreign", reference = "consumers", required = true, on_delete = "cascade" }, },
       { group = { type = "string", required = true } },
+      { tags         = typedefs.tags },
```
Comment (hishamhm) [inline kong/plugins/acl/daos.lua]: style: excess spacing in schema entries? (this and the other `daos.lua` in this PR)

Comment (hbagdi) [reply kong/plugins/acl/daos.lua]: Thanks, fixed!

**Comment by hishamhm** [comment]
@hbagdi Merged! Thanks!

**Pull request merged**